cypress/platform/bundle-test.js

```diff
@@ -1,4 +1,4 @@
-import mermaid from '../../dist/mermaid.core';
+import mermaid from '../../src/mermaid';
```

Comment:
I think this test was originally here to check if the `mermaid.core` file works.

If this isn't testing it anymore, maybe we should delete it/rename/add a comment.

Reply:
I have a suspicion that the current core build might be botched.

- It throws some warnings
- In the mono repo PR, the mindmap package fails to build stating there's no default export.

Reply:
Hmmmm, as a sanity check, we could always run `npm pack` and see if the generated `mermaid.tar.gz` file works with `mermaid-live-editor`.

If it does work, we know the test is broken, but the build works. If it doesn't work, then we know the build is botched, and the test is maybe okay.

Reply:
<img width="287" alt="image" src="https://user-images.githubusercontent.com/10703445/191941999-e99f4cf3-3fe6-426b-afe8-36eaa95803d1.png">

That's a very good test. So it is indeed botched.

Reply:
This was a crazy bug. 
I was specifying esm for core build, but the output was umd. 
Turns out, if you only specify a single output instead of an array, vite will default to building both esm and umd. But as we were specifying core.mjs extension, the umd file was overwriting the esm file. FML.


Also, testing with live editor was a bit tricky. yarn and vite's caches were holding the old umd build and no matter whatever build configs I changed in mermaid-vite, I was getting the same error.

phew 💦

Reply:
> Turns out, if you only specify a single output instead of an array, vite will default to building both esm and umd. But as we were specifying core.mjs extension, the umd file was overwriting the esm file. FML.

:facepalm: That sounds painful. Well, at least the test caught it before we merged this.

> yarn and vite's caches were holding the old umd build

Yep, that happens a lot when using `yarn pack`.

`yarn add git+https://github.com/mermaid-js/mermaid.git#YOUR_BRANCH_OR_GIT_TAG_HERE` usually works better, but unfortunately it doesn't yet support mono-repos.